be/src/olap/memtable.cpp

```diff
@@ -182,17 +182,16 @@ void MemTable::insert(const vectorized::Block* input_block, const std::vector<in
     _mem_usage += input_size;
     _insert_mem_tracker->consume(input_size);
     for (int i = 0; i < num_rows; i++) {
-        _row_in_blocks.emplace_back(new RowInBlock {cursor_in_mutableblock + i});
-        _insert_one_row_from_block(_row_in_blocks.back());
+        _row_in_blocks.emplace(_row_in_blocks.begin(), new RowInBlock {cursor_in_mutableblock + i});
```

Comment:
Why insert elements to the head of `_row_in_blocks`? It's quite low efficient for vector